x/auth/ante/sigverify.go

```diff
@@ -112,18 +110,16 @@ func OnlyLegacyAminoSigners(sigData signing.SignatureData) bool {
 }
 
 func (svd SigVerificationDecorator) VerifyIsOnCurve(pubKey cryptotypes.PubKey) error {
-	if svd.extraVerifyIsOnCurve != nil {
```

Comment:
Please revert this change. It is always nil if you use `NewSigVerificationDecorator` but if you use the `NewSigVerificationDecoratorWithVerifyOnCurve` you can specify one.
This is a feature that just got added (https://github.com/cosmos/cosmos-sdk/pull/23128) :D

Reply:
Ah, didn't realize it's relatively new 🤦‍♂️ 

I just saw that it's always `nil`, so I dropped it.

Reverted:

e69102c